plugins/webp-uploads/helper.php

```diff
@@ -395,3 +395,14 @@ function webp_uploads_sanitize_image_format( string $image_format ): string {
 function webp_uploads_is_picture_element_enabled(): bool {
 	return (bool) get_option( 'webp_uploads_use_picture_element', false );
 }
+
+/**
+ * Checks if the `perflab_generate_webp_and_jpeg` option is enabled.
+ *
+ * @since n.e.x.t
+ *
+ * @return bool True if the option is enabled, false otherwise.
+ */
+function webp_uploads_is_jpeg_fallback_enabled(): bool {
+	return true === (bool) get_option( 'perflab_generate_webp_and_jpeg' );
```

Comment:
Nit to make this less verbose.

```suggestion
	return (bool) get_option( 'perflab_generate_webp_and_jpeg' );
```